### ISSUE TITLE
[Bug]: [GET /v1/provisioning/contact-points] GetContactpoints (status 500): {}

### ISSUE THREAD

**Issue by aryakrishnak** [grafana/terraform-provider-grafana#1606]
### Terraform Version

<2.0.0

### Terraform Grafana Provider Version

<2.8.0

### Grafana Version

10.4.2

### Affected Resource(s)

grafana_contact_point 

### Terraform Configuration Files

```hcl
resource "grafana_contact_point" "asp_grafana_pagerduty" {
  provider = grafana.asp_grafana
  count    = local.pagerduty_enabled ? 1 : 0
  depends_on = [
    pagerduty_service_integration.integration
  ]
  name = var.env_name
  pagerduty {
    integration_key = pagerduty_service_integration.xxxxxx.xxxxx
    component       = var.env_name
    group           = var.env_name
    disable_resolve_message = true
  }
}
```

### Actual Behavior

 │ Error: [GET /v1/provisioning/contact-points] GetContactpoints (status 500): {}
11:36:01  │ 
11:36:01  │   with grafana_contact_point.asp_grafana_pagerduty[0],
11:36:01  │   on mcsp-grafana.tf line 7, in resource "grafana_contact_point" "asp_grafana_pagerduty":
11:36:01  │    7: resource "grafana_contact_point" "asp_grafana_pagerduty" {
11:36:01  │ 
11:36:01  ╵
11:36:01  
11:36:01  Planning failed. Terraform encountered an error while generating this plan.

### Steps to Reproduce

When we did upgrade the grafana version to latest we are getting 500 back, we tried from version greater that 2.7.0 all do have this error >2.0.0,<2.8.0

**Comment by julienduchesne** [comment]
This is the issue in Grafana: https://github.com/grafana/grafana/pull/83070. It was introduced in 10.4 and has unfortunately only been fixed in v11

**Comment by julienduchesne** [comment]
I will probably not be fixing this, since it's fixed in Grafana 11

**Comment by maximusapen** [comment]
@julienduchesne This is going to force people to move to Grafana 11 when they're not exactly ready to make that jump. 10.4 is still supported by Grafana right?

**Comment by julienduchesne** [comment]
> @julienduchesne This is going to force people to move to Grafana 11 when they're not exactly ready to make that jump. 10.4 is still supported by Grafana right?

Yeah. That just reminded me that Amazon Managed Grafana doesn't support Grafana 11. I'll take a look at this

**Comment by julienduchesne** [comment]
This was fixed in v3, but I also backported the fix to v2.19.1
